src/WebJobs.Script.WebHost/Security/Authentication/Keys/AuthenticationLevelHandler.cs

```diff
@@ -38,6 +39,7 @@ internal class AuthenticationLevelHandler : AuthenticationHandler<Authentication
         {
             _secretManagerProvider = secretManagerProvider;
             _isEasyAuthEnabled = environment.IsEasyAuthEnabled();
+            _isStaticWebAppsFunction = environment.IsStaticWebAppsFunction();
```

Comment:
Recommend renaming the check to IsStaticWebApp() since it's an app level thing, not a Function?